README.md

```diff
@@ -1,16 +1,16 @@
-# bedrock-unicode-characters
-Minecraft:Bedrock Edition Unicode characters
+# Bedrock Unicode Characters
+**Minecraft: Bedrock Edition** Unicode characters
 
-## How to use
-Copy the unicode character and paste it into Minecraft: Bedrock Edition chats, signs, titles etc.
+### How to use
+> Copy the unicode character and paste it into Minecraft: Bedrock Edition chats, signs, titles etc.
```

Comment:
I don't even see a reason to use a comment block here at all

Reply:
I've used them a lot in my private repos so I'm kinda used to it but I can remove them!